src/main/webapp/index.html

```diff
@@ -77,6 +77,10 @@
           <span class="sr-only">Loading...</span>
         </div>
       </div>
+      <div id="error-surfacer" style="display:none">
+        <h4 id="error-display-text" style="color:rgb(255, 99, 71);"> Something went wrong :(<br> Try another link? </h4>
+        <h5 id="error-details"style="color:rgb(200, 80, 57);"><h5>
```

Comment:
Can you move style to the css file?
https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns